app/qml/PositionMarker.qml

```diff
@@ -11,6 +11,42 @@ Item {
     property QgsQuick.PositionKit positionKit
     property color baseColor: InputStyle.highlightColor
     property bool withAccuracy: true
+    // animated position
+    property real animatedPosX
+    property real animatedPosY
```

Comment:
would be good to note in comments whether these are screen coordinates or map coordinates